docs/readme.md

```diff
@@ -89,6 +85,14 @@ This will generate the `site` directory which will contain all the gererated doc
 5. Go to your individual virtualization-sdk repo's settings, scroll to the bottom and verify under the GitHub Pages section the `Source` is set to `gh-pages branch`.
 6. Right above this will be a link explaining where your docs are published.
 
+You can also utilize the GitHub workflow for publishing docs (`.github/workflows/publish-docs.yml`). Make sure that the
+workflow is on a branch called `docs/x.y.z`. If you have more than one `docs/x.y.z` branch, you have to push your doc
+changes to the docs branch with the latest `x.y.z` version. Otherwise, the workflow won't run. You also have to make
+sure to enable [publishing from gh-pages branch](https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source).
```

Comment:
can we clarify that we need to set `gh-pages` as the publishing source?

Reply:
I rewrote this paragraph:
"You can also utilize the GitHub workflow for publishing docs (`.github/workflows/publish-docs.yml`) associated with a pull request. Make sure that the workflow is on a branch called `docs/x.y.z` on your fork of the repository. If you have more than one `docs/x.y.z` branch in your fork, you have to push your doc changes to the docs branch with the latest `x.y.z` version. Otherwise, the workflow won't run. You also have to make sure to choose gh-pages branch on your fork as the [publishing source](https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source)."

Reply:
> Make sure that the workflow is on a branch called docs/x.y.z on your fork of the repository.
Could you clarify what this means? Is the workflow only present in `develop` usually? What steps should a developer take, should they create a `docs/1000.0.0` branch from `develop` and just `push` to it which will cause the workflow to get triggered?

Reply:
I updated the paragraph to the following: 

You can also utilize the GitHub workflow for publishing docs (`.github/workflows/publish-docs.yml`) associated with a pull request. The workflow is present on the `develop` branch. Create a branch called `docs/x.y.z` off `develop` on your fork of the repository to ensure that your docs branch triggers the workflow. If you have more than one `docs/x.y.z` branch in your fork, you have to push your doc changes to the docs branch with the latest `x.y.z` version. Otherwise, the workflow won't run. You also have to make sure to choose `gh-pages` branch on your fork as the [publishing source](https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#choosing-a-publishing-source). Once you push doc changes to the `docs/.x.y.z` branch, the docs site should be available under `<your-github-username>.github.io/virtualization-sdk` shortly after. You can see the status of publishing under 
`https://github.com/<your-github-username>/virtualization-sdk/actions`. This is a fast way to give a preview of your changes in a pull request.